Add OpenProse program to keep agent-sdk-ts aligned with upstream Python SDK (#763)
enyst/OpenHands-Tab: .openhands/skills/open-prose/SKILL.md

---
name: open-prose
description: |
  OpenProse is a programming language for AI sessions. An AI session is a Turing-complete
  computer; OpenProse structures English into unambiguous control flow.

  OpenHands-Tab integration:

>   - This repo vendors the OpenProse VM docs as an AgentSkills-format skill.

Author: ```suggestion
  - This repo vendors the OpenProse VM docs as your Skill.
```

---
name: open-prose
description: |
  OpenProse is a programming language for AI sessions. An AI session is a Turing-complete
  computer; OpenProse structures English into unambiguous control flow.

  OpenHands-Tab integration:
  - This repo vendors the OpenProse VM docs as an AgentSkills-format skill.

>   - Project skills are loaded from `<workspaceRoot>/.openhands/skills/` in local mode.

Author: ```suggestion
```
